### DISCUSSION TITLE
status of 406???

### DISCUSSION THREAD

**Discussion by FujiwaraSatomi** [community/community#21259] [Other Feature Feedback, Questions, & Ideas] [answered]
I received a 406 error when trying to open [this repository](https://github.com/yuzupon1133/osu-tools/blob/main/extra/osu_bbcode_parser.js).
but open `osu_bbcode_parser.js` from [here](https://github.com/yuzupon1133/osu-tools/tree/main/extra) and it will display correctly.
why???

*(I got lost in selecting categories)*

**Comment by ronaldkroon** [comment]
It seems direct links to files are broken.

**Reply by Snailedlt** [reply]
Only for script files it seems. That is `.js` and `.json` files. Seems a new update brought the bug a few hours ago. Hopefully it'll be fixed soon

**Comment by FujiwaraSatomi** [comment, marked as answer]
Seems it was a temporary bug.

**Reply by Snailedlt** [reply]
Yeah, it's fixed now :)
